### PR TITLE
Refs [TASK][44588] invest management UI , select invest

### PR DESCRIPTION
**Pull request by vietnh-1109**
Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/44588
https://edu-redmine.sun-asterisk.vn/issues/44587
WHAT
Evidence (Screenshot or Video)
image

Review Checklist
Category	View Point	Description	Self review	Reviewer2 (name)
Conventions	Sun* coding conventions	https://github.com/framgia/coding-standards/tree/master/eng/android	
 yes
 yes
Conventions	Kotlin coding conventions	https://kotlinlang.org/docs/coding-conventions.html	
 yes
 yes
Redmine	Sun* Redmine working process	https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md	
 yes
 yes
Notes (Optional)
(Impacted Areas in Application(List features, api, models or services that this PR will affect))

(List gem, library third party add new)

(Other notes)

### CODE REVIEW

File: lib/bloc/invest_management/invest_management_bloc.dart [hunk 1]
```diff
@@ -8,10 +8,12 @@ import 'package:vn_crypto/ultils/Constant.dart';
 class InvestManagementBloc extends Bloc<InvestManagementEvent, InvestManagementState> {
   final InvestRepository investRepository;
 
-  InvestManagementBloc(this.investRepository) : super(InvestManagementInitialize()) {
+  InvestManagementBloc({required this.investRepository}) : super(InvestManagementInitialize()) {
+    print('preparing...');
```
Comment (ghost) [inline lib/bloc/invest_management/invest_management_bloc.dart]: remove này

File: lib/bloc/invest_management/invest_management_bloc.dart [hunk 2]
```diff
@@ -37,4 +39,16 @@ class InvestManagementBloc extends Bloc<InvestManagementEvent, InvestManagementS
     await investRepository.deleteInvest(itemCoin);
     emit(InvestManagementSuccess(AppStrings.messageDeleteInvestSuccess));
   }
+
+  void _onGetAllCoinMarket(InvestManagementEvent event) async {
+    print('getting');
```
Comment (ghost) [inline lib/bloc/invest_management/invest_management_bloc.dart]: .

File: lib/ui/components/items/InvestItem.dart [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+          child: Image.network(
+            widget.itemCoin.image,
+            width: 35,
+            height: 35,
+          ),
+        ),
+        Column(
+          children: [
+            
+          ],
```
Comment (ghost) [inline lib/ui/components/items/InvestItem.dart]: trong đây không có gì hả Việt ?

Comment (vietnh-1109) [reply lib/ui/components/items/InvestItem.dart]: cái này đang dựng dở :)) để tôi cmt vào

**Review by ghost — APPROVED** [review]

**Review by tuannm-1276 — APPROVED** [review]

**Pull request merged**